Cargo.toml

```diff
@@ -48,3 +48,5 @@ hex = { version = "0.4" }
 sha2 = { version = "0.10" }
 sysinfo = { version = "0.29" }
 glob = { version = "0.3.1" }
+nonempty-collections = { version="0.1.5" }
```

Comment:
Can we split up the cargo.toml to have a [dev-dependencies] section. We can move some of the dependencies (such as the nonempty-collections into that, as those dependencies aren't included in the release build. I'm worried about too many packages bloating the package size.